# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv-cross-internal/tests/shaders/struct.vert
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv-cross-internal/tests/shaders/struct.vert b/third_party/rust/spirv-cross-internal/tests/shaders/struct.vert
deleted file mode 100644
--- a/third_party/rust/spirv-cross-internal/tests/shaders/struct.vert
+++ /dev/null
@@ -1,21 +0,0 @@
-#version 310 es
-
-layout(location = 0) in vec4 a;
-layout(location = 1) in vec4 b;
-layout(location = 2) in vec4 c;
-layout(location = 3) in vec4 d;
-
-struct V
-{
-    vec4 a;
-    vec4 b;
-    vec4 c;
-    vec4 d;
-};
-
-layout(location = 0) out V v;
-
-void main()
-{
-    v = V(a, b, c, d);
-}